docs/source/quickstart/index.rst

```diff
@@ -1,6 +1,30 @@
 Quickstart
 ==========
 
-(In the future, this section will describe how to use the `OpenDP Library`_.)
+.. contents:: |toctitle|
+    :local:
 
-.. _OpenDP Library: https://github.com/opendp/opendp
+Installation
+------------
+
+The easiest way to get started with OpenDP is from Python. Use ``pip`` to install the `opendp <https://pypi.org/project/opendp/>`_ package from PyPI.
+
+.. code-block:: bash
+
+    % pip install opendp
```

Comment:
This percent reminds me of the `%pip` magic command from notebooks. But the percent is not necessary in a terminal. Why is the percent here?

Reply:
That's an Andy-ism. Now that I know sphinx a little better, I think the right thing to do is this:

```
.. prompt:: bash

     pip install opendp
```

That requires the sphinx-prompt plugin, which I'm adding in #270. As part of that, I can come back and tweak this.

Reply:
I had the same thought but let it go (it was copied from the Getting Started page). I typically see `$` instead of `%` for my shell prompt. That said, I like the idea of simply removing it.

Even without the sphinx-prompt plugin (I haven't heard of it), we can still do `.. code-block:: bash`. There are examples on the Logistics page in the Dev Guide.